server-api/installation.md

```diff
@@ -86,15 +90,17 @@ Getting up and running is as easy as 1, 2, 3, 4, 5 ... 6.
    $ yarn
    ```
 
-4. Setup database seeder for local development \(recommended\)
+##Setting up the database seeder for local development \(recommended\)
```

Comment:
Here is a missing space between `##` and `Setting`.

Reply:
will add